.gitignore

```diff
@@ -38,6 +38,9 @@ client/assets/*
 !client/assets/eye-slash.svg
 !client/assets/default/
 !client/assets/fonts/
+client/translations/
+i18n/*
+!i18n/en.po
```

Comment:
shouldn't this be `!i18n/*.po`?
Otherwise we would include only english translations.